src/screens/twinClassFields/twin-class-fields.tsx

```diff
@@ -138,36 +197,43 @@ export function TwinClassFields() {
     toast.success("Class field created successfully!");
   };
 
-  if (!twinClass) {
+  if (!twinClassId) {
     console.error("TwinClassFields: no twin class");
     return;
   }
   return (
     <Experimental_CrudDataTable
+      title="Fields"
       ref={tableRef}
       columns={[
         colDefs.id,
+        colDefs.twinClassId,
         colDefs.key,
         colDefs.name,
         colDefs.description,
+        colDefs.fieldTyperFeaturerId,
+        colDefs.viewPermissionId,
+        colDefs.editPermissionId,
         colDefs.required,
       ]}
       getRowId={(row) => row.key!}
       fetcher={fetchFields}
-      disablePagination={true}
       pageSizes={[10, 20, 50]}
       onRowClick={(row) =>
-        router.push(
-          `/workspace/twinclass/${twinClass!.id!}/twinField/${row.id}`
-        )
+        router.push(`/workspace/twinclass/${twinClassId}/twinField/${row.id}`)
```

Comment:
change to `        router.push(`/workspace/twinclass/${row.twinClassId}/twinField/${row.id}`)`